packages/components/src/models.ts

```diff
@@ -84,138 +56,42 @@ export type PropsInitializer = () => IStateModelProps;
 export function getStateQueryGridModel(
     gridId: string,
     schemaQuery: SchemaQuery,
-    initProps?: IStateModelProps | PropsInitializer,
+    initProps?: IQueryGridModel | PropsInitializer,
     keyValue?: any
 ): QueryGridModel {
     const modelId = getStateModelId(gridId, schemaQuery, keyValue);
 
     // if the model already exists in the global state, return it
     const model = getQueryGridModel(modelId);
+
     if (model) {
         return model;
     }
 
     const metadata = getQueryMetadata();
 
     let modelProps: Partial<IQueryGridModel> = {
-        allowSelection: true,
-        baseFilters: List<Filter.IFilter>(),
-        bindURL: true,
-        containerPath: undefined,
-        containerFilter: undefined,
-        editable: false,
+        keyValue,
```

Comment:
It looks like you're getting rid of some of the default values here (e.g., allowSelection: true).  Do we capture those defaults elsewhere?

Reply:
They are already defined on the QueryGridModel record class :-)

Reply:
I just verified that the removed properties here match the default values set on the QueryGridModel.